Wbooks/src/app/screens/bookDetail/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ScrollView } from 'react-native-gesture-handler';
+
+import bookModel from '../../proptypes/bookModel';
+import reviewModel from '../../proptypes/reviewModel';
+
+import BookInformation from './components/bookInformation';
+import styles from './styles';
+import ReviewsList from './components/reviewsList';
+import reviewsMock from './reviewsmock.json';
+
+function BookDetailContainer({ route }) {
+  const { book } = route.params;
+  const reviews = reviewsMock;
```

Comment:
I think this is not necessary, you can pass `reviewsMock` as the value for `reviews` prop on line 19